explorer/src/main/java/com/scalar/client/tool/explorer/ExplorerException.java

```diff
@@ -21,8 +21,22 @@
 
 package com.scalar.client.tool.explorer;
 
+import com.scalar.dl.ledger.service.StatusCode;
+
 public class ExplorerException extends RuntimeException {
```

Comment:
Is this not used anymore ?

Reply:
Thank you for review comment.
StatusCode used in `registerContract` method.

Reply:
No I mean the exception.

Reply:
`registerContract` is the only place that uses the exception ?
Do we need it ?

Reply:
@feeblefakie san,
I think it is needed because it uses for handling the application side(explorer) exception. ClientException used for handling SDK related exceptions.
It is used to handle the IO exception on the application side.
So I think It's needed for the application.

Could you please confirm

Reply:
OK, it makes  sense. Let's keep it.